.gitignore

```diff
@@ -1,14 +1,20 @@
 #
 .vscode
 
+weights/
+102flowers/
+logs/
+test.py
 # Byte-compiled / optimized / DLL files
 __pycache__/
 *.py[cod]
 *$py.class
 
 # C extensions
 *.so
-
+*.png
+*.jpg
+*.hfc
```

Comment:
Should be removed since we need those files. If you need that in some particular folder create gitignore file there and add whatever is needed.